packages/flutter/test/cupertino/nav_bar_transition_test.dart

```diff
@@ -1229,7 +1221,7 @@ void main() {
     await tester.pump(const Duration(milliseconds: 50));
     expect(
       tester.getTopLeft(flying(tester, find.text('Page 2'))),
-      const Offset(351.00985169410706, 13.5),
+      const Offset(350.0, 13.5),
     );
 
```

Comment:
There may be one already but make sure that there's a test that checks the animation when a back gesture is flung and dropped.